apps/web-giddh/src/app/invoice/proforma/proforma-list.component.html

```diff
@@ -108,13 +89,76 @@
                 </div>
             </div>
         </div>
+        <div class="d-flex mt-1">
+            <div *ngIf="voucherType === 'proformas'" class="btn-wrapper mr-15">
+                <button class="btn btn-pink" style="margin: 0 !important;" (click)="goToInvoice('proformas')">+ New
```

Comment:
remove inline style

Reply:
I am not added this inline css, may be its spacing changes which is created by code formatting.